config/ignition_edifice_debian_buster.yaml

```diff
@@ -2,120 +2,148 @@ name: ignition_edifice_debian_buster
 method: http://packages.osrfoundation.org/gazebo/debian-stable
 suites: [buster]
 component: main
-architectures: [amd64, source]
+architectures: [amd64, armhf, arm64, source]
 filter_formula: "\
-(Package (= ignition-edifice)
-), $Version (% 1.0.2-*) |\
+(Package (= ignition-edifice) \
+), $Version (% 1.0.0-1*) |\
```

Comment:
It's odd that the current repository appears to have regressed in version compared to what was configured previously.

Looking at the ROS 2 repository: http://repo.ros2.org/ubuntu/main/pool/main/i/ignition-edifice/

It does not appear that ignition-edifice on Buster ever had a successful 1.0.2 import since the import job will not downgrade a package.
```
ignition-edifice_1.0.0-1~buster.debian.tar.xz      16-Apr-2021 13:57                1300
ignition-edifice_1.0.0-1~buster.dsc                16-Apr-2021 13:57                1582
ignition-edifice_1.0.0-1~buster_amd64.deb          16-Apr-2021 13:57                2084
```

Reply:
I'm not surprised, when Buster builds fail we usually take a long time to go back and fix them.

> It does not appear that ignition-edifice on Buster ever had a successful 1.0.2 import 

That's one of the reasons I say that these yaml files don't really correspond to the state of the packages. Imports have been failing silently since we started this process 😬

Reply:
The entire purpose of the exercise is to get _closer_.
If I could find a non-exhaustive way to do it I'd fail the import on a missing match.